protected/modules/OphTrOperationbooking/models/Element_OphTrOperationbooking_Operation.php

```diff
@@ -1199,15 +1197,15 @@ public function schedule($booking, $operation_comments, $session_comments, $oper
                 }
             }
         }
-
+        $this->booking->session = $session;
```

Comment:
Does this actually work? I can't imagine that would update the IDs and would therefore not be linked correctly.

Reply:
Currently it seems work, but I suspect as well. Maybe keep observing this part.

Reply:
@marshl What are your thoughts on this?

Reply:
Setting the session_id might only be necessary for saving, not for validation. It does seem a bit weird though.